### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by pavel-gutsal**
https://pavel-gutsal.github.io/react_static-list-of-todos/

### CODE REVIEW

File: src/components/TodoInfo/TodoInfo.tsx [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
+    className={classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': todo.completed,
+      },
+    )}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+    <UserInfo user={todo.user} />
+  </article>
```
Comment (valerii-derkach) [inline src/components/TodoInfo/TodoInfo.tsx:23]: render user only if it is exists

**Review by valerii-derkach — CHANGES_REQUESTED** [review]

**Comment by pavel-gutsal** [comment]
user will be rendered only if  property value is truish

File: src/components/TodoInfo/TodoInfo.tsx [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+};
+
+export const TodoInfo: React.FC<Prop> = ({ todo }) => (
+  <article
+    className={classNames(
+      'TodoInfo',
+      {
+        'TodoInfo--completed': todo.completed,
+      },
+    )}
```
Comment (Bohdan-Maksymiuk) [inline src/components/TodoInfo/TodoInfo.tsx:19]: use `classnames` library for conditional classes
https://www.npmjs.com/package/classnames

File: src/components/TodoInfo/TodoInfo.tsx [hunk 3]
```diff
@@ -1,4 +1,24 @@
...
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+    { todo?.user && (<UserInfo user={todo.user} />)}
```
Comment (Bohdan-Maksymiuk) [inline src/components/TodoInfo/TodoInfo.tsx:22]:
```suggestion
    {todo?.user && <UserInfo user={todo.user} />}
```
or if component is big
```
  {todo?.user && (
    <UserInfo user={todo.user} />
  )}
```

**Review by Bohdan-Maksymiuk — APPROVED** [review]
Good job 👍

**Review by alexandra-protyanova — APPROVED** [review]

**Pull request closed without merging**